kaldb/src/main/java/com/slack/kaldb/chunkManager/ChunkRollOverStrategy.java

```diff
@@ -1,6 +1,12 @@
 package com.slack.kaldb.chunkManager;
```

Comment:
Please move chunk roll over strategies to their own package like chunkrollover in the kaldb folder.